config/tutorials/otel-logs/index-config.yaml

```diff
@@ -5,30 +5,68 @@
 
```

Comment:
actually, this log index config file is used in a tutorial that does not use the otel schema. It's a simpler use case. Maybe we should review the tutorial so that we ingest to the otel index directly?

Reply:
I have not been able to spot the tutorial you are referring to.

Reply:
I am updating that tutorial here https://github.com/quickwit-oss/quickwit/pull/3534

Reply:
Cool! Can you revert the index config update then in this PR and do it in #3534 ?

Reply:
Yes that's what i did to avoid conflicts only the tutorials files are now changed

Reply:
But I still see changes here. In the tutorial, I think we should not create the index and use the OTEL one created by default. So we can remove this index config file.